client/src/components/signup/style.css

```diff
@@ -0,0 +1,20 @@
+form{
```

Comment:
this will change the style for all forms and `h3`'s we have,
give it a className and style based on it